pkg/kube/kube.go

```diff
@@ -47,8 +47,8 @@ type Kube interface {
 	FluxPresent(ctx context.Context) (bool, error)
 	GetClusterName(ctx context.Context) (string, error)
 	GetClusterStatus(ctx context.Context) ClusterStatus
-	GetApplication(ctx context.Context, name string) (*wego.Application, error)
 	LabelExistsInCluster(ctx context.Context, label string) error
+	GetApplication(ctx context.Context, name, namespace string) (*wego.Application, error)
```

Comment:
How about using [`types.NamespacedName`](https://pkg.go.dev/k8s.io/apimachinery/pkg/types#NamespacedName) which is more strongly typed version of this?

Reply:
Much better :+1: